.github/workflows/framework-tests.yaml

```
@@ -37,7 +37,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        python-version: [3.5, 3.6, 3.7, 3.8, 3.9]
+        python-version: [3.5, 3.6, 3.8, "3.10"]
```

Good spotting to add quotes on the `3.10` for YAML's sake. Let's use quotes for all of the list items for consistency:

```suggestion
        python-version: ["3.5", "3.6", "3.8", "3.10"]
```